.changeset/rotten-monkeys-tie.md

```diff
@@ -0,0 +1,6 @@
+---
+"near-api-js": patch
```

Comment:
Only the changed package should be indicated here. `changeset` will bump versions of packages with a dependency on the changed packages at publish time.

Reply:
oops, I must have misclicked